static/javascripts/components/questions.jsx

```diff
@@ -209,17 +209,8 @@ getInitialState: function(){
 * textfield and get it to POST
 */
 var FreeResponse = React.createClass({
-
     getInitialState: function(){
-        var responseState = this.props.responseState;
-        var responseStateLength = Object.keys(responseState).length;
-        var prevAnswer = "Change Me!";
-        for(var i = responseStateLength-1; i >= 0; i--){
-          if(responseState[i].question_id == this.props.questionID){
-             prevAnswer = responseState[i].response_data;
-          }
-        }
-        return {answer: prevAnswer};
```

Comment:
This code was written in order to preserve previous answers to open response. So it probably is an antipattern but we'll have to find some alternative so we're not losing functionality.